src/model/BaseballMaker.js

```diff
@@ -0,0 +1,50 @@
+import { pickRandomNumberInRange } from '../utils/random.js';
+
+class BaseballMaker {
+  /**
+   * @static
+   * @public
+   * @type {import('../utils/jsDoc').BaseballShape}
+   */
+  static BASEBALL_SHAPE = Object.freeze({
+    minNumber: 1,
+    maxNumber: 9,
+    size: 3,
+  });
+
+  /**
+   * @private
+   * @type {import('../utils/jsDoc').BaseballShape}
+   */
+  #baseballShape;
+
+  constructor() {
+    this.#baseballShape = BaseballMaker.BASEBALL_SHAPE;
+  }
+
+  /**
+   * BaseballMaker 클래스의 정적 팩토리 메서드
+   * @static
+   * @public
+   * @returns {BaseballMaker} BaseballMaker의 인스턴스
+   */
+  static create() {
+    return new BaseballMaker();
+  }
+
+  /**
+   * @public
+   * @returns {number[]} 야구공
+   */
+  createBaseball() {
+    const baseball = new Set();
+    const { minNumber, maxNumber } = this.#baseballShape;
+    while (baseball.size < this.#baseballShape.size) {
+      const baseballDigit = pickRandomNumberInRange(minNumber, maxNumber);
+      baseball.add(baseballDigit);
+    }
+    return [...baseball];
```

Comment:
```suggestion
    const baseball = new Set();
    const { minNumber, maxNumber } = this.#baseballShape;
    
    while (baseball.size < this.#baseballShape.size) {
      const baseballDigit = pickRandomNumberInRange(minNumber, maxNumber);
      baseball.add(baseballDigit);
    }
    return [...baseball];
```
P5 - 위에도 개행으로 가독성을 챙겨보자는 의견을 주신 분이 있지만, 저 또한 비슷한 의견이에요!
단순히 선언부-기능부를 개행으로 분리하는 것 만으로도 가독성이 좋아진다고 생각합니다.

다만, 개인적으로 while문 내부처럼 너무 짧으면 저는 그냥 붙여버리는 편 입니다 (개인적으로는 3줄까지는 붙이고 4줄부터는 분리해주는 것 같아요)

Reply:
이번 공통 피드백으로 개행 얘기도 나와서 다음 미션 땐 적극적으로 사용해보겠습니다!! 감사해요 :)